docs/ibc/integration.md

```diff
@@ -104,14 +104,14 @@ func NewApp(...args) *App {
   app.CapabilityKeeper = capabilitykeeper.NewKeeper(appCodec, keys[capabilitytypes.StoreKey], memKeys[capabilitytypes.MemStoreKey])
```

Comment:
I've updated integration.md to use the new package alias

cc @damiannolan  / @crodriguezvega

Reply:
Just a nit, but I think it would be nice: could you please add a note somewhere that `ibcexported` is an import alias for `"github.com/cosmos/ibc-go/v6/modules/core/exported"`? At the moment somebody looking at this pseudocode wouldn't know where that's coming from...